lightning/src/util/enforcing_trait_impls.rs

```diff
@@ -259,13 +270,13 @@ impl Writeable for EnforcingSigner {
 impl EnforcingSigner {
 	fn verify_counterparty_commitment_tx<'a, T: secp256k1::Signing + secp256k1::Verification>(&self, commitment_tx: &'a CommitmentTransaction, secp_ctx: &Secp256k1<T>) -> TrustedCommitmentTransaction<'a> {
 		commitment_tx.verify(&self.inner.get_channel_parameters().as_counterparty_broadcastable(),
-		                     self.inner.counterparty_pubkeys(), self.inner.pubkeys(), secp_ctx)
```

Comment:
nit: maybe ok to remove?

Reply:
Ah, fair. 

This is just my editor fixing the inconsistent use of spaces and tabs here. I think `rust-lightning` uses tabs, but it's not enforced.

Reply:
I removed it!